File: prime_num.java
import java.io.*;
import java.lang.*;
import java.util.Scanner;

class prime
{
public static void main (String[] args){
        
        Scanner sc= new Scanner(System.in);  
        int num = sc.nextInt(); 
    
        if(num > 1)
        {
        //check for factors
        for(int i = 2;i<num; i++)
        {
        if((num % i) == 0)
        {
           System.out.println("is not a prime number");
           break;
        }
        else
        {
            System.out.println("is a prime number");
            break;
        }
        }
        }
        else
        {
            System.out.println("is not a prime number");
        }
}
}

File: primeNum.cs
using System;

class PrimeNumber{
    // function check whether a
    // number is prime or not
    static bool isPrime(int n)
    {
        // Corner case
        if (n <= 1)
            return false;
 
        // Check from 2 to n-1
        for (int i = 2; i < n; i++)
            if (n % i == 0)
                return false;
 
        return true;
    }
 
    // Driver Code
    static void Main()
    {
        if (isPrime(11))
            Console.Write("given number is a prime number");
 
        else
            Console.Write("given number is not a prime number");
    }
}